gpbft/messagetemplate.go

```diff
@@ -0,0 +1,97 @@
+package gpbft
+
+import (
+	"math/big"
+
+	xerrors "golang.org/x/xerrors"
+)
+
+type MessageTemplate struct {
+	powerTable powerTableAccess
+
+	NetworkName NetworkName
+	Payload     Payload
+
+	BeaconForTicket []byte
```

Comment:
This came from the host too. Maybe we can avoid plumbing this through too. Should be able to look it up by instance. (The sim host probably doesn't have this mapping yet).